vocode/streaming/synthesizer/eleven_labs_synthesizer.py

```diff
@@ -48,50 +48,44 @@ def __init__(
         self.words_per_minute = 150
         self.experimental_streaming = synthesizer_config.experimental_streaming
 
-        # Create a PydubWorker instance as an attribute
-        if self.experimental_streaming:
-            self.miniaudio_worker = MiniaudioWorker(
-                synthesizer_config, asyncio.Queue(), asyncio.Queue()
-            )
-            self.miniaudio_worker.start()
-
-    async def tear_down(self):
-        await super().tear_down()
-        if self.experimental_streaming:
-            self.miniaudio_worker.terminate()
-
     async def output_generator(
         self,
         response: aiohttp.ClientResponse,
         chunk_size: int,
         create_speech_span: Optional[Span],
     ) -> AsyncGenerator[SynthesisResult.ChunkResult, None]:
+        # Create a PydubWorker instance as an attribute
+        miniaudio_worker = MiniaudioWorker(
+            self.synthesizer_config, chunk_size, asyncio.Queue(), asyncio.Queue()
+        )
+        miniaudio_worker.start()
```

Comment:
Can't we just do miniaudio_worker = MiniaudioWorker(
            self.synthesizer_config, chunk_size, asyncio.Queue(), asyncio.Queue()
        ).start()?

Reply:
no because `start` returns the worker task, not the worker